src/cloudai/runner/kubernetes/kubernetes_runner.py

```diff
@@ -48,7 +48,7 @@ def _submit_test(self, tr: TestRun) -> KubernetesJob:
             k8s_system: KubernetesSystem = cast(KubernetesSystem, self.system)
             job_name, job_namespace = k8s_system.create_job(job_spec)
 
-        return KubernetesJob(self.mode, self.system, tr, job_namespace, job_name, job_kind, job_output_path)
```

Comment:
Since we are passing `tr` as an argument, can we avoid passing `tr.output_path`? Would that make sense? Same for other runners.